docker/README.md

```diff
@@ -0,0 +1,62 @@
+# Docker Images #
+
+The following docker images have been verified to successfully build the source-build repo by:
+
+1. Building source-build
+2. Building the resulting tarball
+3. Running smoke tests
+
+| OS | Image Name | Link to Dockerfile | Verification Date |
```

Comment:
Commit hash should be included IMO. Also, I think the table would look cleaner with the Dockerfile link just added to an existing column, and I think the link should be pinned to the image's commit, like:

| OS | Image Name |  Tested commit |
| --- | --- | --- |
| CentOS 7.1 | [`microsoft/dotnet-buildtools-prereqs:centos-7-b46d863-20180719033416`](https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/b46d863/src/centos/7/Dockerfile) | [4f3eacd](https://github.com/dotnet/source-build/commit/4f3eacdd0551677444ab917d4e332653672e0bd5), 2018.09.05

Reply:
The commit hash is in the image name, so I didn't think it was necessary in the link.

Reply:
I think it makes it easy to generate the link, but I think it's needed or else people won't notice they're being linked to a mismatched Dockerfile.

Reply:
I see what you're saying.  Updated.